packages/card/src/Card.story.tsx

```diff
@@ -9,6 +9,11 @@ import LGCard, { CardProps } from './Card';
 // https://github.com/storybookjs/storybook/issues/15334
 // eslint-disable-next-line react/jsx-props-no-spreading
 type CardStoryProps = BoxProps<'div', CardProps>;
+
+// TODO: Import below comment directly from component definition.
```

Comment:
What does this mean?

Reply:
My understanding is that the component description on L15 below should be defined in `Card.tsx`, but can't currently for TS trickery reasons?

Reply:
There is currently a copy of the component in the `.story.tsx` file. This was defined as a copy because Storybook isn't reading the component's props correctly for some reason (i.e. it's a known issue that Storybook doesn't correctly process props when components use `forwardRef`). 

Because the component was redefined, it also tries to read the TSDoc from the component's copy instead of the original component. So the TSDoc itself also had to be copied over.

Once the issues around `forwardRef` are resolved, this component copy and its TSDoc should be removed so there aren't 2 copies of the same TSDoc.